### DISCUSSION TITLE
Ability to generate contributor reports for a category

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#11734] [Ideas] [answered]
I can generate reports for all my project, or for a component.

But I need to generate them for a category.

I use a category to unify all the little components that are part of Tor Browser, and I want to have a report of the translations for that category only.

**Comment by nijel** [comment, marked as answer]
Turned into a feature request: https://github.com/WeblateOrg/weblate/issues/11739
